netcontrold/lib/util.py

```diff
@@ -63,7 +63,7 @@ def cpuinfo():
         regex = re.compile('^(.*?)\s*:\s*(.*)')
         (param, val) = regex.match(line).groups()
 
-        if param == 'processor' and val == '':
+        if (param == 'processor' and val == '') or (param == 'core id' and val == '') or (param == 'physical id' and val == ''):
```

Comment:
Say which value can not be null .. "Value %s cannot be null" % param ?

Reply:
ok